site/content/internal/infrastructure/onelogin-aws.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Onelogin and AWS"
+date: 2018-11-07T16:10:15+01:00
+weight: 45
+---
+
+# Generating Keys using Onelogin authentication
+
+To get temporary AWS Keys to use we are using this project [https://github.com/physera/onelogin-aws-cli](https://github.com/physera/onelogin-aws-cli)
+
+## Installation
+
+To install, use pip:
+
+```Bash
+$ pip install onelogin-aws-cli
+```
+
+Note that `onelogin-aws-cli` requires Python 3.
+
+Note that it is not recommended to install Python packages globally
+on your system.
+
+## Usage
+
+Running `onelogin-aws-login`  will perform the authentication against OneLogin,
```

Comment:
Needs more details here. How to get the client ID / secret and app id are not clear. 
Also this will only work for OneLogin admins.

Reply:
i will check this later this week

Reply:
managed to make it work, was missing one thing in the eks side. will update the docs

Reply:
done